packages/commonwealth/server/routes/canvas.ts

```diff
@@ -0,0 +1,69 @@
+import { AppError } from 'common-common/src/errors';
+import type { DB } from '../models';
+import type { TypedRequestBody, TypedResponse } from '../types';
+import { success } from '../types';
+import * as Sequelize from 'sequelize';
+
+type CanvasDataReq = {
+  query: {
+    before: string
+  }
+};
+type CanvasDataResp = {};
```

Comment:
Undefined type or actually empty?

Reply:
This should be an empty object for now, to be filled in later.